packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -487,7 +487,8 @@ export class ElectronMainApplication {
 
     protected async handleStopRequest(electronWindow: BrowserWindow, onSafeCallback: () => unknown, reason: StopReason): Promise<void> {
         // Only confirm close to windows that have loaded our front end.
-        const safeToClose = !electronWindow.webContents.getURL().includes(this.globals.THEIA_FRONTEND_HTML_PATH) || await this.checkSafeToStop(electronWindow, reason);
+        const frontendPath = this.globals.THEIA_FRONTEND_HTML_PATH.replace(/\\/g, '/');
+        const safeToClose = !electronWindow.webContents.getURL().includes(frontendPath) || await this.checkSafeToStop(electronWindow, reason);
```

Comment:
Should we perform the same normalization on the URL for safety's sake?

Reply:
Sure, I don't see any harm in it 👍